parsers/PE.py

```diff
@@ -32,8 +32,8 @@ def fetch_production(zone_key='PE', session=None, target_datetime=None, logger=N
     r = session or requests.session()
     url = 'http://www.coes.org.pe/Portal/portalinformacion/Generacion'
     response = r.post(url, data={
-        'fechaInicial': arrow.now(tz=tz).format('DD/MM/YYYY'),
-        'fechaFinal': arrow.now(tz=tz).replace(days=+1).format('DD/MM/YYYY'),
+        'fechaInicial': arrow.now(tz=tz).replace(days=-1).format('DD/MM/YYYY'),
```

Comment:
We need to use `.shift()` here.

```shell
>>> import arrow
>>> arw=arrow.utcnow()
>>> arw
<Arrow [2020-01-03T12:41:19.470793+00:00]>
>>> arw.replace(days=-1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/chris/arrow/arrow/arrow.py", line 622, in replace
    raise AttributeError('unknown attribute: "{}"'.format(key))
AttributeError: unknown attribute: "days"
>>> arw.shift(days=-1)
<Arrow [2020-01-02T12:41:19.470793+00:00]>
```

Reply:
Not sure what version of arrow is installed on your machine Alex but you can update to the latest dependencies with `pip install -r requirements.txt` from the parser directory.

Reply:
Thanks @systemcatch, I upgraded my arrow and fixed it! ;)